script/test

```diff
@@ -1,10 +1,21 @@
 #!/bin/bash -e
 
-SERVICES="atst authnid fundz"
-cd $(dirname $0)/../services
+cd $(dirname $0)/..
+source services.ini
+cd services
```

Comment:
This won't work if the directory doesn't exist yet, right? We need to either create the directory, or not git ignore it.

Reply:
Ah... I see it's in the other script... hard to tell, maybe a comment here?

Reply:
Aaah... yeah, that'll work.